### ISSUE TITLE
Graphicsmagick v1.3.30 : throws exception 

### ISSUE THREAD

**Issue by dimarusyy** [Microsoft/vcpkg#3951]
Trying to run simple test (borrowed from http://www.graphicsmagick.org/Magick++/Image.html) with the following code (x86-windows-static triplet) :

```
#include <Magick++.h>
#include <iostream>
using namespace std;
using namespace Magick;
int main(int argc,char **argv)
{
  // Initialize the API.  Can pass NULL if argv is not available.
  InitializeMagick(*argv);

  // Construct the image object. Seperating image construction from the
  // the read operation ensures that a failure to read the image file
  // doesn't render the image object useless.
  Image image;

  try {
    // Determine if Warning exceptions are thrown.
    // Use is optional.  Set to true to block Warning exceptions.
    image.quiet( false );

    // Read a file into image object
    image.read( "girl.gif" );

    // Crop the image to specified size (width, height, xOffset, yOffset)
    image.crop( Geometry(100,100, 100, 100) );

    // Write the image to a file
    image.write( "x.gif" );
  }
  catch( Exception &error_ )
    {
      cout << "Caught exception: " << error_.what() << endl;
      return 1;
    }
  return 0;
}

```

gives the following error :
> exception: test: Unable to access configuration file (delegates.mgk) reported by d:\dev\vcpkg\buildtrees\graphicsmagick\src\graphicsmagick-1.3.30-windows-source\magick\
> blob.c:2135 (GetConfigureBlob)

Setting 
> set MAGICK_CONFIGURE_PATH=D:\dev\vcpkg\installed\x86-windows-static\share\graphicsmagick\config

doesn't help. 

Any suggestions ?
Thanks

**Comment by ras0219-msft** [comment]
Thanks for posting this issue! Is this resolved by #4215?

**Comment by JackBoosY** [comment]
Hi @dimarusyy, if you still have problem with this, please create a new issue.
Thanks.
